granary/meetup.py

```diff
@@ -26,6 +26,21 @@ class Meetup(source.Source):
             domain=DOMAIN,
             approve=EVENT_URL_RE)
 
+    @classmethod
+    def embed_post(cls, obj):
+        """Returns the HTML string for embedding an RSVP from Meetup.com.
+
+        Args:
+          obj:
+          obj: AS1 dict with at least url, and optionally also content.
+
+        Returns: string, HTML
+        """
+        return '<span class="verb">RSVP %s</span> to <a href="%s">this event</a>.' % (
```

Comment:
[ugh i think i lost my intitial version of this comment in a previous commit]

does meetup have any HTML snippets for embedding events? ie rendering a user-visible preview of them? we do that in most (all?) publish previews so that users can confirm they're publishing what they mean to publish. not strictly necessary for deploying live, but would be nice eventually if possible.

Reply:
> does meetup have any HTML snippets for embedding events? ie rendering a user-visible preview of them? we do that in most (all?) publish previews so that users can confirm they're publishing what they mean to publish. not strictly necessary for deploying live, but would be nice eventually if possible.

on second thought, don't worry about this, it's fine as is.

Reply:
Doesn't look like it out-of-the-box but good thinking. Shall I raise an issue to track this for the future?

Reply:
up to you! i'm ok either way.

Reply:
OK, I'll leave it for now then